fbinternal/cloud/go/services/testcontroller/statemachines/enodebd.go

```diff
@@ -339,8 +339,30 @@ States with traffic tests enabled:
 		> "Check for upgrade" pings slack, 1 minute
 */
 
-func startNewTest(*enodebdE2ETestStateMachine, *models.EnodebdTestConfig) (string, time.Duration, error) {
-	return checkForUpgradeState, time.Minute, nil
+func startNewTest(machine *enodebdE2ETestStateMachine, config *models.EnodebdTestConfig) (string, time.Duration, error) {
+	states := []string{
+		checkForUpgradeState,
```

Comment:
Let's reduce this to just states that modify some parameters (i.e check for upgrade, rebootEnodeb1State, reconfigEnodebSTate, restoreEnodebConfigState, subscriberInactive, subscriberActive). Don't include the retry states as well (e.g. trafficTest1State2, trafficTest1State3)